### DISCUSSION TITLE
If the user chooses to use Apple authentication by web3 auth, how can we get the user's walletAddress.

### DISCUSSION THREAD

**Discussion by HidetakaKo** [Web3Auth/Web3Auth#673] [Help]
If the user chooses to use Apple authentication by web3 auth, how can we get the user's walletAddress.

Will we have to get them separately after login?

**Comment by rinzler99** [comment]
Hi, You can follow below steps to login with apple and get walletAddress:
1. Use web3auth sdk. 
2. configure web3auth modal to use only apple login. [ref](https://web3auth.io/docs/whitelabel/login-modal)
3. once user logs in, get user account with getAccounts() method. [ref](https://web3auth.io/docs/sdk/web/providers/evm#web3ethgetaccounts)
